Service/PrometheusService.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Modules\ItkPrometheus\Service;
+
+use Prometheus\CollectorRegistry;
+use Prometheus\RenderTextFormat;
+
+class PrometheusService
+{
+  /**
+   * @var \Prometheus\CollectorRegistry
+   */
+  private CollectorRegistry $collectorRegistry;
+
+
+  /**
+   * @param \Prometheus\CollectorRegistry $registry
+   */
+  public function __construct (CollectorRegistry $registry)
+  {
+    $this->collectorRegistry = $registry->getDefault();
+  }
+
+  /**
+   * @return string
```

Comment:
Function comment would be nice to have. I think it renders the metrics end-point for scraping ?